### DISCUSSION TITLE
Idea: Allow ignoring built-in libraries to use custom versions of them

### DISCUSSION THREAD

**Discussion by j-px** [Sketchware-Pro/Sketchware-Pro#696] [Ideas]
Currently, using e.g. a newer Firebase library version isn't possible unless you don't use Firebase Components and/or Blocks. The files of the built-in library are always used instead of Local libraries'. An option to stop that would be great.

This could for example be added to Library Manager, maybe at the bottom of the screen, and between it and Firebase, AdMob, etc. settings, empty space.

**Comment by j-px** [comment]
Implemented! If anyone wants to test that feature out, get a snapshot build built today or later.

**Reply by AndroXStudiO** [reply]
It will work for any library?  Not for only firebase?? I mean if i used NetworkRequest component,  then i can use newer version of okkhttp as local libs??

**Reply by AndroXStudiO** [reply]
> Implemented! If anyone wants to test that feature out, get a snapshot build built today or later.


Testing DONE!😍 its really working fine😍😍🥰

**Reply by j-px** [reply]
> It will work for any library? Not for only firebase?? I mean if i used NetworkRequest component, then i can use newer version of okkhttp as local libs??

Any built-in library can be excluded with this, so yeah, you can replace the version of OkHttp Sketchware Pro automatically adds to projects.

**Reply by abrihamdemisse** [reply]
Just a stupid question, I'm a beginner on github, where do i find the snapshot files

**Reply by j-px** [reply]
[Here, the Actions tab while filtering for pushes to branch `main`](https://github.com/Sketchware-Pro/Sketchware-Pro/actions?query=event%3Apush+branch%3Amain)

**Reply by abrihamdemisse** [reply]
so, where is the link, please send the download link of the apk that this feature has beed implemented on. i couldn't figure it out?

**Reply by j-px** [reply]
Click on one of the CI builds there, then scroll to Artifacts.

**Comment by PranavPurwar** [comment]
OMG PROOOOO
